rpc-protocol/src/main/java/org/sonarsource/sonarlint/core/rpc/protocol/client/fix/FileEditDto.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * SonarLint Core - RPC Protocol
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.rpc.protocol.client.fix;
+
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileEditDto {
+
+  private final Path path;
+  private final List<ChangesDto> changes;
+
+  public FileEditDto(Path path, List<ChangesDto> changes) {
+    this.path = path;
```

Comment:
can we have clearer name for this? Is it IDE file path? Or full file path? or server path?

Reply:
I will rename this to `idePath` 👍🏻